### DISCUSSION TITLE
An option to limit a set only in private windows

### DISCUSSION THREAD

**Discussion by DelusionalOptimist** [proginosko/LeechBlockNG#357] [Ideas]
Before anything else, I'd like to thank the maintainers. I've personally found the extension to be really helpful in limiting the time I spend procrastinating : )

**The problem**
A user might be using a particular site for productive work in general but find themselves procrastinating in private mode.
Thus, they might want to block access to a site only when running in private mode.

**Possible solution**
Having a checkbox in the set rules which specifies if they want to block it only in private windows. When checked, leechblock will detect if that site is accessed in a private context ([Reference](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extension/inIncognitoContext)) and block accordingly.

**Comment by proginosko** [comment]
Good suggestion. Should be easy to implement.

**Comment by proginosko** [comment]
Added in version 1.6.2.

**Comment by DelusionalOptimist** [comment]
Ooh nice... Thank you so much for this @proginosko! :heart:
